photos.html

```diff
@@ -0,0 +1,109 @@
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+
+      <main class="main-content">
+        <section class="preamble">
+          <div class="preamble__inner">
+            <h2 class="preamble__header">Photo and video</h2>
+            <p class="preamble__paragraph">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
+          </div>
+        </section>
+
+        <section class="gallery__main-photo">
+          <a class="gallery_main-img"></a>
+          <div class="gallery__inner">
+            <h3 class="gallery__title">Outback</h3>
+            <p class="gallery__author">Photo by: Boris</p>
+            <p class="gallery__likes">1350</p>
+          </div>
+        </section>
+
+        <section class="gallery__collection">
+          <section class="gallery__photo-vegetation">
+            <a class="gallery_vegetation-img"></a>
+            <div class="gallery__inner">
+              <h3 class="gallery__title">Local vegetation</h3>
+              <p class="gallery__author">Photo by: Sergey</p>
+              <p class="gallery__likes">143</p>
+            </div>
+          </section>
+
+          <section class="gallery__photo-road">
+            <a class="gallery_road-img"></a>
+            <div class="gallery__inner">
+              <h3 class="gallery__title">Road north</h3>
+              <p class="gallery__author">Photo by: Peter</p>
+              <p class="gallery__likes">96</p>
+            </div>
+          </section>
+
+          <section class="gallery__photo-bridge">
+            <a class="gallery_bridge-img"></a>
+            <div class="gallery__inner">
+              <h3 class="gallery__title">The Devil's Bridge</h3>
+              <p class="gallery__author">Photo by: Anton</p>
+              <p class="gallery__likes">254</p>
+            </div>
+          </section>
+        </section>
+
+        <section class="gallery__video">
+          <div class="video__inner">
+            <h2 class="gallery__header-video">Still not convinced?</h2>
+            <p class="gallery__paragraph-video">See the video presentation and more for tickets, until they have risen in price again!</p>
+            <video calss="gallery__video-source" src="" controls preload="auto" poster="">
+              <p class="gallery__vide--error">Your browser doesn't support HTML5 video tag.</p>
+            </video>
```

Comment:
at the moment don't know how we're going to style player controls, but it looks like we'll have to create our own controls with js. which could be a cool thing. https://developer.mozilla.org/en-US/Apps/Fundamentals/Audio_and_video_delivery/cross_browser_video_player